src/main/java/nextstep/subway/domain/Line.java

```diff
@@ -219,4 +229,21 @@ public void updateLine(LineRequest lineRequest) {
             this.color = lineRequest.getColor();
         }
     }
+
+    private void addSectionToLine(Section section) {
+        sections.add(section);
+        section.setLine(this);
+    }
+
+    private void removeMiddleSection(Station station) {
+        Section targetSection = sections.stream().filter(s -> s.getDownStation().getId().equals(station.getId()))
+                .findFirst()
+                .orElseThrow(() -> new CustomException(CustomException.NOT_EXIST_STATION_IN_LINE));
+
+        Section nextSection = nextSection(station);
+        nextSection.setUpStation(targetSection.getUpStation());
+        nextSection.plusDistance(targetSection.getDistance());
+
+        sections.remove(targetSection);
```

Comment:
```suggestion
        Section previousSection = getPreviousSection(station);
        Section nextSection = nextSection(station);
        nextSection.update(previousSection.getUpStation, previousSection.getDistance);
        remove(previousSection)
```

target이라는 명보다는 previous와 같이 명확한 이름이 좋지 않을까요~?